### PR TITLE
Revert "Update related_website_sets.JSON"

### PR DESCRIPTION
**Pull request by Airborn22**
Reverts GoogleChrome/related-website-sets#371

Reason: https://github.com/GoogleChrome/related-website-sets/pull/371#issuecomment-2103432161

### CODE REVIEW

**Review by cfredric — APPROVED** [review]

**Review by ratchanonsuttawas — COMMENTED** [review]
>Review these changes

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: >review request 4da4f9ca4fa8b2cfb525be8b59da92ed620db28a

**Comment by ratchanonsuttawas** [comment]
> Reverts GoogleChrome/related-website-sets#371
> 
> Reason: https://github.com/GoogleChrome/related-website-sets/pull/371#issuecomment-2103432161

4da4f9ca4fa8b2cfb525be8b59da92ed620db28a

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > >review request 4da4f9ca4fa8b2cfb525be8b59da92ed620db28a

**Review by ratchanonsuttawas — COMMENTED** [review]
Review these changes

**Comment by ratchanonsuttawas** [comment]
> 
> 
> It appears you have failed some tests. Here are your results:
> The set associated with https://sapo.pt was removed from the list, but https://sapo.pt/.well-known/related-website-set.json does not return error 404.
>

**Comment by ratchanonsuttawas** [comment]
> Reverts GoogleChrome/related-website-sets#371
> 
> Reason: https://github.com/GoogleChrome/related-website-sets/pull/371#issuecomment-2103432161

**Pull request merged**